src/main/scala/devices/tilelink/Plic.scala

```diff
@@ -229,7 +231,8 @@ class TLPLIC(params: PLICParams, beatBytes: Int)(implicit p: Parameters) extends
     }
 
     def thresholdRegDesc(i: Int) = RegFieldDesc(s"threshold_$i", s"Interrupt & claim threshold for target $i. Maximum value is ${nPriorities}.",
-      reset=if (nPriorities > 0) None else Some(1), access=RegFieldAccessType.RWSPECIAL)
+      reset=if (nPriorities > 0) None else Some(1),
```

Comment:
Threshold,  priority, and enable registers are _not_ reset.

Reply:
right, but if there is only one priority value they are constant 1, right? That's what the code is trying to say.

Reply:
oooh I misread the code. ya